packages/rocketchat-slackbridge/slackbridge.js

```diff
@@ -1047,20 +1075,45 @@ class SlackBridge {
 
 				if (rocketMsgObj) {
 					RocketChat.deleteMessage(rocketMsgObj, rocketUser);
+					logger.class.debug('Rocket message deleted by Slack');
 				}
 			}
 		}
 	}
 
 	/*
+	 https://api.slack.com/events/message/message_changed
+	 */
+	processSlackMessageChanged(slackMessage) {
+		if (slackMessage.previous_message) {
+			let currentMsg = RocketChat.models.Messages.findOneById(this.createRocketID(slackMessage, slackMessage.message.ts));
+
+			//Only process this change, if its an actual update (not just Slack repeating back our Rocket original change)
+			if (currentMsg && (slackMessage.message.text !== currentMsg.msg)) {
+				let rocketChannel = this.getRocketChannel(slackMessage);
+				let rocketUser = slackMessage.previous_message.user ? this.findRocketUser(slackMessage.previous_message.user) || this.addRocketUser(slackMessage.previous_message.user) : null;
+
+				let rocketMsgObj = {
+					//@TODO _id
+					_id: this.createRocketID(slackMessage, slackMessage.previous_message.ts),
+					rid: rocketChannel._id,
+					msg: this.convertSlackMsgTxtToRocketTxtFormat(slackMessage.message.text),
+					updatedBySlack: true	//We don't want to notify slack about this change since Slack initiated it
```

Comment:
I assume it is OK to temporarily add a state flag (updatedBySlack) so that I can key off of it when later fired back to me via Rocket?